src/utils.ts

```diff
@@ -96,6 +100,8 @@ interface HardwareValidation {
  */
 export async function validateHardware(): Promise<HardwareValidation> {
   try {
+    if (DesktopConfig.devCpuMode) return { isValid: true, gpu: 'cpu' };
```

Comment:
Let's change the gpu wording in the PR to `device` instead as the value can be `cpu` as well.